src/test/scala/com/databricks/spark/csv/util/InferSchemaSuite.scala

```diff
@@ -52,4 +83,14 @@ class InferSchemaSuite extends FunSuite {
       Array(LongType)).deep == Array(DoubleType).deep)
   }
 
+  test("Type/Schema inference works as expected for the simple parse dataset.")
```

Comment:
No, I think it is okay though. I just said this because I see it has been added in this way.

For example, https://github.com/databricks/spark-csv/pull/235 and https://github.com/databricks/spark-csv/pull/224